src/test/java/org/minions/devfund/mauricioz/ActorTest.java

```diff
@@ -0,0 +1,86 @@
+package org.minions.devfund.mauricioz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class is a set of unit tests for Actor class methods.
+ *
+ * @author Mauricio Zelaya
+ * @version 1.0
+ */
+public class ActorTest {
+    private Actor actor;
+    private Movie mov1;
+    private Movie mov2;
+    private List<Movie> movies;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movies" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)